### PR TITLE
FOUR-20443: The permission "View All Cases" and "View My Request" are not showing

### PR DESCRIPTION
**Pull request by pmPaulis**
## Issue & Reproduction Steps
The permission "View All Cases" and "View My Request" are not showing

## Solution
- List the changes you've introduced to solve the issue.

## How to Test
Describe how to test that this solution works.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-20443

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-67f41f0cf4.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-67f41f0cf4.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
The following permissions need to create:
**View All Cases**
**View My Request** (this enable per default)

Captura de pantalla 2024-11-22 a las 3 11 43 p  m

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-67f41f0cf4.engk8s.processmaker.net

File: upgrades/2024_11_19_161132_create_permission_view_all_cases_all_requests.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        ],
+    ];
+
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        // Update the group Requests to Cases and Requests
+        DB::table('permissions')
```
Comment (caleeli) [inline upgrades/2024_11_19_161132_create_permission_view_all_cases_all_requests.php:27]: Consider to verify the `view-all_requests` has the right label `View All Requests`

**Comment by caleeli** [comment]
**Note:** Seem the fresh install script in kubernetes is right and executes the `php artisan upgrade`
https://github.com/ProcessMaker/pm4-k8s-distribution/blob/release-2024-fall/images/entrypoints/ent.installpm4.entrypoint.sh

But the documentation link at READEM.md is not updated, since in the documentation link:
https://processmaker.atlassian.net/wiki/spaces/PM4/pages/480149598/Server+Deployment

Does not include the execution of `php artisan upgrade`, consider to create a documentation ticket to add the missing command.

File: upgrades/2024_11_19_161132_create_permission_view_all_cases_all_requests.php (hunk 1 above)
Comment (pmPaulis) [reply upgrades/2024_11_19_161132_create_permission_view_all_cases_all_requests.php:27]: @caleeli the observation was fixed

**Review by caleeli — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-67f41f0cf4.engk8s.processmaker.net

**Pull request merged**